Makefile

```diff
@@ -57,3 +54,29 @@ package: all rss
 lint:
 	pre-commit --version > /dev/null || python3 -m pip install pre-commit
 	pre-commit run --all-files
+
+# New Sphinx targets:
+
+SPHINX_JOBS=8
```

Comment:
Hi @AA-Turner 

Why 8?

Reply:
Arbitrary number, beyond that diminishing returns. This was tested on GHA CI infrastructure, so a different number may be faster locally.

I'm considering disabiling search capabilities, as people will likely be better served with `site:peps.python.org blah` searches, and I think that the word stemming process takes a bit of time. Sphinx builds on CI took around 7 minutes, which does seem quite long (3-4x straight docutils)